core/domain/app_feedback_report_constants.py

```diff
@@ -133,7 +133,7 @@ class AndroidNetworkType(enum.Enum):
     NONE = 'none'
 
 
-FILTER_FIELD_NAMES = app_feedback_report_models.FILTER_FIELD_NAMES
+FILTER_FIELD_NAMES = app_feedback_report_models.FilterFieldNames
```

Comment:
Should this not follow the same convention (i.e. ``FilterFieldNames``)? It makes the value references below inconsistent with the direct reference versions (e.g. ``FilterFieldNames.REPORT_TYPE`` vs. ``FILTER_FIELD_NAMES.REPORT_TYPE``).

Reply:
Since it was only referencing the enum and wasn't actually a enum I had chosen to leave the casing for it unchanged but I do see the benefit of it so I'll make the changes.

Reply:
Done